### PR TITLE
feat: Rename Open Y to WS

### PR DESCRIPTION
**Pull request by podarok**
Addresses: https://yusa.atlassian.net/browse/DS-241

### CODE REVIEW

File: modules/openy_demo_addthis/openy_demo_addthis.info.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-name: Open Y Demo AddThis
+name: YMCA Website Services Demo AddThis
 description: Enables AddThis service demo functionality.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_addthis/openy_demo_addthis.info.yml]: lets also delete this
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_ahb/openy_demo_ahb.info.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
-name: Open Y Demo Advanced Help Blocks
+name: YMCA Website Services Demo Advanced Help Blocks
 description: Stores migrations to import demo Advanced help blocks.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_ahb/openy_demo_ahb.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_bamenities/openy_demo_bamenities.info.yml [hunk 3]
```diff
@@ -1,9 +1,9 @@
-name: Open Y Demo Block Amenities
+name: YMCA Website Services Demo Block Amenities
 description: Stores migrations to import demo Amenities blocks.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_bamenities/openy_demo_bamenities.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_bmicrosites_menu/openy_demo_bmicrosites_menu.info.yml [hunk 4]
```diff
@@ -1,9 +1,9 @@
-name: Open Y Demo Block Microsites Menu
+name: YMCA Website Services Demo Block Microsites Menu
 description: Stores migrations to import demo MIcrosites menu.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_bmicrosites_menu/openy_demo_bmicrosites_menu.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_bsimple/openy_demo_bsimple.info.yml [hunk 5]
```diff
@@ -1,9 +1,9 @@
-name: 'Open Y Demo Block Simple'
+name: 'YMCA Website Services Demo Block Simple'
 description: Stores migrations to import demo Simple blocks.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_bsimple/openy_demo_bsimple.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_menu/modules/openy_demo_menu_footer/openy_demo_menu_footer.info.yml [hunk 6]
```diff
@@ -1,8 +1,8 @@
-name: 'Open Y Demo Menu Footer'
+name: 'YMCA Website Services Demo Menu Footer'
 description: Stores migrations to import demo Footer menu.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_menu/modules/openy_demo_menu_footer/openy_demo_menu_footer.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_menu/modules/openy_demo_menu_main/openy_demo_menu_main.info.yml [hunk 7]
```diff
@@ -1,8 +1,8 @@
-name: Open Y Demo Menu Main
+name: YMCA Website Services Demo Menu Main
 description: Stores migrations to import demo Main menu.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_menu/modules/openy_demo_menu_main/openy_demo_menu_main.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_menu/openy_demo_menu.info.yml [hunk 8]
```diff
@@ -1,9 +1,9 @@
-name: Open Y Demo Menu
+name: YMCA Website Services Demo Menu
 description: Stores migration group for demo menu migrations.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_menu/openy_demo_menu.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_nalert/openy_demo_nalert.info.yml [hunk 9]
```diff
@@ -1,9 +1,9 @@
-name: 'Open Y Demo Node Alert'
+name: 'YMCA Website Services Demo Node Alert'
 description: Stores migrations to import demo alerts.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_nalert/openy_demo_nalert.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_nblog/openy_demo_nblog.info.yml [hunk 10]
```diff
@@ -1,9 +1,9 @@
-name: 'Open Y Demo Node Blog'
+name: 'YMCA Website Services Demo Node Blog'
 description: Stores migrations to import demo blog posts.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_nblog/openy_demo_nblog.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_nbranch/openy_demo_nbranch.info.yml [hunk 11]
```diff
@@ -1,9 +1,9 @@
-name: 'Open Y Demo Node Branch'
+name: 'YMCA Website Services Demo Node Branch'
 description: Stores migrations to import demo branch nodes.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_nbranch/openy_demo_nbranch.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_ncamp/openy_demo_ncamp.info.yml [hunk 12]
```diff
@@ -1,9 +1,9 @@
-name: 'Open Y Demo Node Camp'
+name: 'YMCA Website Services Demo Node Camp'
 description: Stores migrations to import demo camp nodes.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_ncamp/openy_demo_ncamp.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_nclass/openy_demo_nclass.info.yml [hunk 13]
```diff
@@ -1,9 +1,9 @@
-name: 'Open Y Demo Node Class'
+name: 'YMCA Website Services Demo Node Class'
 description: Stores migrations to import demo class nodes.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_nclass/openy_demo_nclass.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_nevent/openy_demo_nevent.info.yml [hunk 14]
```diff
@@ -1,9 +1,9 @@
-name: Open Y Demo Node Event
+name: YMCA Website Services Demo Node Event
 description: Stores migrations to import demo event nodes.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_nevent/openy_demo_nevent.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_nfacility/openy_demo_nfacility.info.yml [hunk 15]
```diff
@@ -1,9 +1,9 @@
-name: 'Open Y Demo Node Facility'
+name: 'YMCA Website Services Demo Node Facility'
 description: Stores migrations to import demo facility nodes.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_nfacility/openy_demo_nfacility.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_nhome_alt/openy_demo_nhome_alt.info.yml [hunk 16]
```diff
@@ -1,9 +1,9 @@
-name: Open Y Demo Node Home Alternative
+name: YMCA Website Services Demo Node Home Alternative
 description: Stores migrations to import alternative demo home page when demo content disabled.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_nhome_alt/openy_demo_nhome_alt.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_nlanding/openy_demo_nlanding.info.yml [hunk 17]
```diff
@@ -1,9 +1,9 @@
-name: 'Open Y Demo Node Landing Page'
+name: 'YMCA Website Services Demo Node Landing Page'
 description: Stores migrations to import demo landing pages.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_nlanding/openy_demo_nlanding.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_nmbrshp/openy_demo_nmbrshp.info.yml [hunk 18]
```diff
@@ -1,9 +1,9 @@
-name: Open Y Demo Node Membership
+name: YMCA Website Services Demo Node Membership
 description: Stores migrations to import demo membership nodes.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_nmbrshp/openy_demo_nmbrshp.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_nnews/openy_demo_nnews.info.yml [hunk 19]
```diff
@@ -1,9 +1,9 @@
-name: Open Y Demo Node News
+name: YMCA Website Services Demo Node News
 description: Stores migrations to import demo news nodes.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_nnews/openy_demo_nnews.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_nprogram/openy_demo_nprogram.info.yml [hunk 20]
```diff
@@ -1,9 +1,9 @@
-name: 'Open Y Demo Node Program'
+name: 'YMCA Website Services Demo Node Program'
 description: Stores migrations to create demo program subcategory nodes.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_nprogram/openy_demo_nprogram.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_nsessions/openy_demo_nsessions.info.yml [hunk 21]
```diff
@@ -1,9 +1,9 @@
-name: Open Y Demo Node Sessions
+name: YMCA Website Services Demo Node Sessions
 description: Stores migrations to create demo session nodes.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.2
```
Comment (anpolimus) [inline modules/openy_demo_nsessions/openy_demo_nsessions.info.yml]:
```suggestion
version: 9.x-1.2
```

File: modules/openy_demo_nsocial_post/openy_demo_nsocial_post.info.yml [hunk 22]
```diff
@@ -1,9 +1,9 @@
-name: Open Y Demo Node Social Post
+name: YMCA Website Services Demo Node Social Post
 description: Stores migrations to create demo social post nodes.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_nsocial_post/openy_demo_nsocial_post.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_tamenities/openy_demo_tamenities.info.yml [hunk 23]
```diff
@@ -1,9 +1,9 @@
-name: Open Y Demo Taxonomy Amenities
+name: YMCA Website Services Demo Taxonomy Amenities
 description: Stores migrations to import demo amenities taxonomy terms.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_tamenities/openy_demo_tamenities.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_tarea/openy_demo_tarea.info.yml [hunk 24]
```diff
@@ -1,9 +1,9 @@
-name: Open Y Demo Taxonomy Area
+name: YMCA Website Services Demo Taxonomy Area
 description: Stores migrations to import demo area taxonomy terms.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_tarea/openy_demo_tarea.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_tblog/openy_demo_tblog.info.yml [hunk 25]
```diff
@@ -1,9 +1,9 @@
-name: Open Y Demo Taxonomy Blog Category
+name: YMCA Website Services Demo Taxonomy Blog Category
 description: Stores migrations to import demo blog category taxonomy terms.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_tblog/openy_demo_tblog.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_tfacility/openy_demo_tfacility.info.yml [hunk 26]
```diff
@@ -1,9 +1,9 @@
-name: Open Y Demo Taxonomy Facility Type
+name: YMCA Website Services Demo Taxonomy Facility Type
 description: Stores migrations to import demo facility type taxonomy terms.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_tfacility/openy_demo_tfacility.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_tfitness/openy_demo_tfitness.info.yml [hunk 27]
```diff
@@ -1,9 +1,9 @@
-name: Open Y Demo Taxonomy Fitness Category
+name: YMCA Website Services Demo Taxonomy Fitness Category
 description: Stores migrations to import demo fitness category taxonomy terms.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0-rc1
```
Comment (anpolimus) [inline modules/openy_demo_tfitness/openy_demo_tfitness.info.yml]:
```suggestion
version: 9.x-1.0-rc1
```

File: modules/openy_demo_tnews/openy_demo_tnews.info.yml [hunk 28]
```diff
@@ -1,9 +1,9 @@
-name: Open Y Demo Taxonomy News Category
+name: YMCA Website Services Demo Taxonomy News Category
 description: Stores migrations to import demo news category taxonomy terms.
 type: module
 core_version_requirement: ^8 || ^9
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_tnews/openy_demo_tnews.info.yml]:
```suggestion
version: 9.x-1.0
```

File: modules/openy_demo_webform/openy_demo_webform.info.yml [hunk 29]
```diff
@@ -1,8 +1,8 @@
-name: Open Y Demo Webform
+name: YMCA Website Services Demo Webform
 description: Provides a demo webform.
 type: module
 core_version_requirement: ^8 || ^9
-package: Open Y
+package: YMCA Website Services
 version: 8.x-1.0
```
Comment (anpolimus) [inline modules/openy_demo_webform/openy_demo_webform.info.yml]:
```suggestion
version: 9.x-1.0
```

**Comment by froboy** [comment]
Cursor_and_Basic_site_settings___Test

@podarok I was able to install this and get demo content which all looked good. The only issue was that the Default front page didn't seem to get set properly. I can't find where that is though.

File: modules/openy_demo_nhome_alt/config/install/migrate_plus.migration.openy_demo_lp_paragraph_home_alt_simple.yml [hunk 30]
```diff
@@ -16,8 +16,8 @@ source:
       id: home_alt_simple
       parent_id: about
       description: |
-        <h1>Explore the Open Y!</h1>
-        <p>Learn how to use the Open Y content management system to deliver brand-compliant digital experiences.<br />
+        <h1>Explore the YMCA Website Services!</h1>
```
Comment (froboy) [inline modules/openy_demo_nhome_alt/config/install/migrate_plus.migration.openy_demo_lp_paragraph_home_alt_simple.yml:19]:
```suggestion
        <h1>Explore YMCA Website Services!</h1>
```

File: modules/openy_demo_nhome_alt/config/install/migrate_plus.migration.openy_demo_lp_paragraph_home_alt_simple.yml [hunk 31]
```diff
@@ -16,8 +16,8 @@ source:
...
+        <p>Learn how to use the YMCA Website Services content management system to deliver brand-compliant digital experiences.<br />
```
Comment (froboy) [inline modules/openy_demo_nhome_alt/config/install/migrate_plus.migration.openy_demo_lp_paragraph_home_alt_simple.yml]:
```suggestion
        <h1>Explore YMCA Website Services!</h1>
        <p>Learn how to use YMCA Website Services content management system to deliver brand-compliant digital experiences.<br />
```

File: modules/openy_demo_nlanding/config/install/migrate_plus.migration.openy_demo_lp_paragraph_banner.yml [hunk 32]
```diff
@@ -18,12 +18,12 @@ source:
     -
       id: homepage_banner
       parent_id: homepage
-      headline: 'Explore the Open Y!'
+      headline: 'Explore the YMCA Website Services!'
```
Comment (froboy) [inline modules/openy_demo_nlanding/config/install/migrate_plus.migration.openy_demo_lp_paragraph_banner.yml]:
```suggestion
      headline: 'Explore YMCA Website Services!'
```

**Review by froboy — CHANGES_REQUESTED** [review]
Fix a couple spots.

**Pull request merged**